package.json

```diff
@@ -69,7 +75,8 @@
     "build:prod": "npm run test && npm run build",
     "build:deploy": "webpack --progress --colors --mode=deploy --target=web",
     "watch": "webpack --progress --colors --watch --mode=deploy --target=web",
-    "test": "node_modules/.bin/mocha tests/**/*.test.* --require tests/setup.js",
+    "est": "node_modules/.bin/mocha tests/**/*.test.* --require tests/setup.js",
```

Comment:
I had not seen this, you deleted a 't' in 'test'. Please readd it.

Reply:
that was intentional, the actual 'test' is jest, i think if you scroll down some more, you would see it